remoting/pom.xml

```diff
@@ -28,8 +28,8 @@
     <name>rocketmq-remoting ${project.version}</name>
 
     <properties>
-        <maven.compiler.source>1.6</maven.compiler.source>
-        <maven.compiler.target>1.6</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
Do not modify compiler level. we need to keep to be compatible with JDK 1.6 in SDK

Reply:
I used some features in JDK 1.8. I'll try to resolve this compatible problem ASAP.